### PR TITLE
migration(group): backfill Ignored Groups with substatus null values

### PR DESCRIPTION
**Pull request by barkbarkimashark**
Partially undoes this PR https://github.com/getsentry/sentry/pull/47101 for Ignored Groups.

The default for Group.Ignored should have been null and not `UNTIL_ESCALATING`

### CODE REVIEW

File: tests/sentry/migrations/test_0419_backfill_groupedmessage_ignored_substatus.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from sentry.models import Group, GroupStatus, GroupSubStatus
+from sentry.testutils.cases import TestMigrations
+
+
+class GroupIgnoredSubstatusBackfillTest(TestMigrations):
+    migrate_from = "0418_add_actor_constraints"
+    migrate_to = "0419_backfill_groupedmessage_ignored_substatus"
+
+    def setup_before_migration(self, apps):
+        self.create_group(status=GroupStatus.IGNORED, substatus=GroupSubStatus.UNTIL_ESCALATING),
```
Comment (armenzg) [inline tests/sentry/migrations/test_0419_backfill_groupedmessage_ignored_substatus.py:10]: Why is there a comma after the line? I've not noticed this before in Python.

File: tests/sentry/migrations/test_0419_backfill_groupedmessage_ignored_substatus.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        self.create_group(status=GroupStatus.MUTED, substatus=GroupSubStatus.UNTIL_ESCALATING),
+
+        # below substates shouldn't be legal but for the purposes of this backfill, we'll null them out
+        self.create_group(status=GroupStatus.IGNORED, substatus=GroupSubStatus.ONGOING),
```
Comment (armenzg) [inline tests/sentry/migrations/test_0419_backfill_groupedmessage_ignored_substatus.py:14]: What happens when create_group is called with bad values? Does it actually create the group? Should it raise an error?

Comment (barkbarkimashark) [reply tests/sentry/migrations/test_0419_backfill_groupedmessage_ignored_substatus.py]: When we start enforcing the bad value substatuses, this should definitely throw an error. But at the current state of master, it'll simply allow illegal substate values. 
See this PR https://github.com/getsentry/sentry/pull/47103
I closed it, but will create another PR with the proper validations once this PR is merged.

**Review by armenzg — APPROVED** [review]
🎉

**Pull request merged**